src/uct/ib/base/ib_iface.h

```diff
@@ -205,6 +205,7 @@ struct uct_ib_iface {
         unsigned              tx_max_poll;
         unsigned              seg_size;
         unsigned              roce_path_factor;
+        uint8_t               max_inl_recv;
```

Comment:
some refactoring suggestion:
1. define here uint8_t max_inline[UCT_IB_DIR_NUM]
2. pass UCT_IB_DIR_TX/RX to uct_ib_iface_create_cq(), instead of *cq, and *inl. uct_ib_iface_create_cq() will use "iface" directly, and take the fields according to "dir"
3. remove local variable "inl" from ib_iface constructor

Reply:
max_inl_recv[DIR]